your-rents-services-geodata/src/main/java/com/yourrents/services/geodata/exception/DataNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.yourrents.services.geodata.exception;
+
+public class DataNotFoundException extends RuntimeException{
+
+	public DataNotFoundException(String message) {
+		super(message);
+	}
+}
```

Comment:
Add at least a constructor with two parameters: message and "cause" throwable.

...for supporting exception concatenation.

Reply:
sure, I planned to add at that time it was needed, but I will change it.